activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+*   Add ability to change transaction isolation for all pools within a block.
+
+    If your application needs to change transaction isolation for the duration of a
+    block, use `ActiveRecord::Base.with_transaction_isolation_level {}`. This will change
```

Comment:
If this is fully global, perhaps it would be better to define it on `ActiveRecord` rather than `ActiveRecord::Base`.

This way it doesn't look like it's a per model or per db_config action.

Reply:
Good point, I'll move it tomorrow.